docs/_documentations/mdt-intellij-getting-started.md

```diff
@@ -41,19 +48,20 @@ Complete the prerequisites and choose to install Codewind on IntelliJ with eithe
 4. Click the template of your choice. Then, click **Next**.
 5. Enter the **Project name:** field to name your project and enter the **Project location:** field to choose where to put it.
 6. Click **Finish**. A window for your project appears.
-7. IntelliJ detects a `pom.xml` file and asks if you want to add the file as a Maven project. In the **Non-managed pom.xml file found:** window, click **Add as Maven Project** to access your project source files and create the correct project structure.
-   - The **Non-managed pom.xml file found:** window might disappear after some time. To open it again, go to the IntelliJ project view and click the `pom.xml` file. From the context menu, click **Add as Maven Project** to create the correct project structure.
+7. IntelliJ detects a `pom.xml` file and asks if you want to add the file as a Maven project. In the **Non-managed pom.xml file found:** notification, click **Add as Maven Project** to access your project source files. From these files, create the correct project structure.
```

Comment:
Clicking **Add as Maven Project** doesn't give access to project source files, the user already has access, and they don't have to do anything other than click the link to set up the correct project structure.  Basically what clicking on this link does is cause IntelliJ to analyze the pom.xml file so it knows how the project is laid out -- i.e. what folders contain the source, what folders contain tests, what folders contain resources etc. (but IntelliJ users will already know that)

So `... click **Add as Maven Project** to access your project source files. From these files, create the correct project structure`
needs to change back to something like the previous
`...click **Add as Maven Project** to create the correct project structure`
or maybe
`...click **Add as Maven Project** which will configure the correct project structure.`